_posts/2025-06-10-mid-senior-engineer.html

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: "Mid-Senior Engineer: A Journey of Self-Discovery"
+date: 2025-06-10
+author_name: Irina Kamalova
+author_role:
+blurb_img: /assets/images/blog/2025-06-10-mid-to-senior-engineer-banner.png
+blurb_img_source:
+description: |
+  So, you've mastered the fundamentals, you're a reliable contributor, and the "Mid-Level
+  Engineer" title feels comfortable. But the ambition to grow, to take on more responsibility,
+  and to reach that coveted "Senior Engineer" status is bubbling beneath the surface. You're
+  not alone! The journey from Mid to Senior is a significant step in any tech professional's
+  career, and understanding the available pathways is crucial for strategic advancement.
+category: Career-progression
+---
+
+<div class="text-justify">
+  <p>
+    <i>So, you've mastered the fundamentals, you're a reliable contributor, and the "Mid-Level
+      Engineer" title feels comfortable. But the ambition to grow, to take on more responsibility,
+      and to reach that coveted "Senior Engineer" status is bubbling beneath the surface. You're
+      not alone! The journey from Mid to Senior is a significant step in any tech professional's
+      career, and understanding the available pathways is crucial for strategic advancement.</i>
+  </p>
+
+  <p>
+    I suggest to look at two strategies following the chart:
+  </p>
+
+  <div class="article-media">
+    <img class="img-fluid" src="/assets/images/blog/2025-06-10-mid-to-senior-engineer.png"
+      alt="mid-to-senior engineer chart">
+  </div>
+
+  <p>
+    <b>Path 1: Get a promotion within your company </b>
+  <ul>
+    <li><b>Goal: </b>Get a promotion within your company. </li>
```

Comment:
```suggestion
    <b>Goal: </b>Get a promotion within your company.
```
I think it is better to keep the goal just bold without bulletpoint.